<ui version="4.0" >
 <class>GardenProperties</class>
 <widget class="QWidget" name="GardenProperties" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Garden Properties</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>&amp;Background</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>&amp;Image:</string>
           </property>
           <property name="buddy" >
            <cstring>backgroundImageEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QLineEdit" name="backgroundImageEdit" >
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="browseBackgroundButton" >
             <property name="text" >
              <string>&amp;Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>&amp;Scale:</string>
           </property>
           <property name="buddy" >
            <cstring>scaleSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="scaleSpinBox" >
           <property name="prefix" >
            <string/>
           </property>
           <property name="suffix" >
            <string>x</string>
           </property>
           <property name="singleStep" >
            <double>0.100000000000000</double>
           </property>
           <property name="value" >
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>&amp;Opacity:</string>
           </property>
           <property name="buddy" >
            <cstring>opacitySlider</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="opacitySlider" >
           <property name="maximum" >
            <number>255</number>
           </property>
           <property name="sliderPosition" >
            <number>255</number>
           </property>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedAppearance" >
            <bool>false</bool>
           </property>
           <property name="invertedControls" >
            <bool>false</bool>
           </property>
           <property name="tickPosition" >
            <enum>QSlider::TicksAbove</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>backgroundImageEdit</tabstop>
  <tabstop>browseBackgroundButton</tabstop>
  <tabstop>scaleSpinBox</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
